civihr_employee_portal/src/View/StaffDirectoryView.php

```diff
@@ -16,13 +16,19 @@ public function alter($view, $query) {
     $queryParts = [];
     $this->getWhereFields($query->where, $queryParts);
     $departmentField = 'hrjc_role_hrjc_revision.role_department';
+    $titleField = 'hrjc_role_hrjc_revision.role_title';
     $roleEndField = 'hrjc_role_hrjc_revision.role_end_date';
+    $roleStartField = 'hrjc_role_hrjc_revision.role_start_date';
 
-    // if department is set we must limit to only current roles
-    if (!empty($queryParts[$departmentField])) {
-      $group = $query->set_where_group('OR'); // add a new WHERE group
-      $query->add_where($group, $roleEndField, date('Y-m-d'), '>=');
-      $query->add_where($group, $roleEndField, NULL, 'IS');
+    // if department or title is set we must limit to only current roles
+    if (!empty($queryParts[$departmentField]) ||
+      !empty($queryParts[$titleField])
+    ) {
+      $orGroup = $query->set_where_group('OR'); // add a new WHERE group
```

Comment:
I think this is `$orGroup` because it's `endDate is in future OR is null`, but you have `$andGroup` that doesn't contain AND conditions. Do you think it would be clearer to rename these `roleEndGroup` and `roleStartGroup` or `roleEndCondition` or something like that?

Reply:
The `add_where` function requires a group which will determine what section the query will be added (AND/OR).  It makes sense to rename as suggested 👌